packages/fluentui/react-northstar-prototypes/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@fluentui/react-northstar-prototypes",
+  "description": "A set of prototypes built using @fluentui/react-northstar",
+  "version": "0.51.0",
+  "bugs": "https://github.com/microsoft/fluentui/issues",
+  "private": true,
+  "dependencies": {
+    "@babel/runtime": "^7.10.4",
+    "@fluentui/accessibility": "^0.51.0",
+    "@fluentui/code-sandbox": "^0.51.0",
+    "@fluentui/docs-components": "^0.51.0",
+    "@fluentui/keyboard-key": "^0.2.12",
+    "@fluentui/react-bindings": "^0.51.0",
+    "@fluentui/react-component-event-listener": "^0.51.0",
+    "@fluentui/react-component-ref": "^0.51.0",
+    "@fluentui/react-icons-northstar": "^0.51.0",
+    "@fluentui/react-northstar": "^0.51.0",
+    "tslib": "^1.10.0",
+    "classnames": "^2.2.6",
+    "copy-to-clipboard": "^3.2.0",
+    "csstype": "^2.6.7",
+    "faker": "^4.1.0",
+    "formik": "2.0.3",
+    "lodash": "^4.17.15",
+    "moment": "^2.18.1",
+    "react": "16.8.6",
+    "react-custom-scrollbars": "^4.2.1",
+    "react-dom": "16.8.6",
+    "react-hook-form": "^5.7.2",
+    "react-textarea-autosize": "7.0.0",
+    "react-virtualized": "^9.21.1",
+    "react-window": "^1.8.6"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta",
+    "@fluentui/scripts": "^1.0.0",
+    "@types/classnames": "^2.2.9",
+    "@types/faker": "^4.1.3",
+    "@types/react": "16.8.25",
+    "@types/react-custom-scrollbars": "^4.0.5",
+    "@types/react-dom": "16.8.4",
+    "@types/react-virtualized": "^9.21.8",
+    "@types/react-window": "^1.8.2",
+    "gulp": "^4.0.2",
+    "typescript": "3.7.2"
+  },
+  "files": [
+    "dist"
+  ],
```

Comment:
Is this required ? our webpack configs should just directly alias the raw src

Reply:
similarly for `jsnext:main`, `main`, `module`

Reply:
Maybe not? I added there only because other packages have it, not because it makes any sense :P 
@layershifter WDYT?

Reply:
As we are using this in E2E tests and we use there CommonJS because of Jest we should keep `main` and `module`. `jsnext:main` can be removed for sure.